HIP/hip-0000-schedule-service-system-contract.md

```diff
@@ -0,0 +1,108 @@
+
+This is a step back in the Hedera UX that was made easier by the use of in transaction signatures.
+
+## Rationale
+
+By providing a secure mechanism to acquire asynchronous authorization from multiple accounts, smart contracts can continue to be used for more decentralized operations whiles still maintaining the integrity of account sovereignty by allowing accounts to approve confirm their participation in a transaction.
+
+## User stories
+
+1. As an EOA I would like to initiate a smart contract transaction that schedules a supported transaction.
+2. As an EOA I would like to initiate a smart contract transaction that allows me to sign a scheduled transaction.
+3. As an EOA I would like to initiate a smart contract transaction that prompts a contract to authorize a scheduled transaction.
+4. As an EOA I would like to initiate a smart contract transaction that allows me to extract information about a scheduled transaction.
+  
+## Specification
+
+The ledger HSCS will utilize the existing scheduled transaction service supported on the ledger within the system contract logic. 
+
+To achieve this a new Hedera Schedule Service (HSS) system contract will need to be created to encompass and expose the necessary scheduled transaction features.
+
+### Hedera Schedule Service (HSS) system contract
+
+A new `IHederaScheduleService` interface will be implemented to allow accounts to authorize a pre-existing scheduled transaction via smart contracts
+
+|   Hash        |   Selector                                                                                        |   In contract support |
+|---------------|---------------------------------------------------------------------------------------------------|-----------------------|
+| `0xf0637961`  | `authorizeSchedule(address) external returns (int64 responseCode)`                                | Y                     |
+| `0x5e147101`  | `getScheduledTransactionInfo(address) external returns (bytes memory transactionProtobufBytes)`   | Y                     |
+| `0xd797b304`  | `signSchedule(address) external returns (int64 responseCode)`                                     | N                     |
+| `0x358eeb03`  | `signSchedule(address, bytes) external returns (int64 responseCode)`                              | Y                     |
+
+Since `signSchedule(address scheduleAddress) returns (int64 responseCode)` relies on an implicit signature it will only callable by EOA’s via the IHRC facade.
+In this case the signature will be the inner ECDSA signature found in the RLP encoded `EthereumTransaction`. 
+For `Contract` and `ContractCreate`  any applicable signature found in the signature map will be utilized as in `ScheduleSign`.
```

Comment:
I'm not sure if I understand what this sentence means?  What is `ScheduleSign` referencing?